public/theme/boltcms-2021/showcase.twig

```diff
@@ -0,0 +1,157 @@
+{% extends 'partials/_master.twig' %}
```

Comment:
Le'ts keep this file _somewhere_, but remove it from this PR. I'd prefer to have the new functionality separate

Reply:
Check